src/main/groovy/org/beryx/runtime/data/TargetPlatform.groovy

```diff
@@ -16,60 +16,42 @@
 package org.beryx.runtime.data
 
 
-import groovy.transform.CompileStatic
 import org.beryx.runtime.util.JdkUtil
+import org.gradle.api.Named
 import org.gradle.api.Project
+import org.gradle.api.internal.provider.DefaultProvider
 import org.gradle.api.logging.Logger
 import org.gradle.api.logging.Logging
+import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
 
```

Comment:
Was there a particular reason for removing `@CompileStatic`?

Reply:
[e61b4c3](https://github.com/beryx/badass-runtime-plugin/pull/140/commits/e61b4c39cbdf8bc323a7155e8499ee7754165cd8) nope, this was me misunderstanding what it was for (I assumed it was similar to `JvmStatic` in Kotlin), added back